barterdude/hooks/metrics/prometheus/senders.py

```diff
@@ -0,0 +1,18 @@
+try:
+    from prometheus_client.metrics import MetricWrapperBase
+except ImportError:
+    print("""
+    Please install extra dependency with:
+        `pip install barterdude[prometheus]`
+    """)
+
+
+class Senders(dict):
```

Comment:
```suggestion
class ImutableDict(dict):
```

Reply:
I don't like this name... Because the value is a prometheus metric... What do you think about change to Metrics?